Exceptions-Debugging.rmd

```diff
@@ -399,8 +399,13 @@ In R, there are three tools for handling conditions (including errors) programma
 * `tryCatch()` lets you specify __handler__ functions that control what
   happens when a condition is signalled.
 
-* `withCallingHandlers()` is a variant of `tryCatch()` that runs its handlers
-  in a different context. It's rarely needed, but is useful to be aware of.
+* `withCallingHandlers()` is a variant of `tryCatch()` that establishes local
+  handlers, whereas `tryCatch()` registers exiting handlers. Local handlers are
+  called in the same context as where the condition is signalled, without
+  interrupting the execution of the function. When a exiting handler from
```

Comment:
I like this terminology. Is there any way to register a exiting handler from `withCallingHandlers()`?